rpm/poppler.spec

```diff
@@ -140,7 +141,13 @@ rm -f %{buildroot}%{_libdir}/*.la
 %{_libdir}/pkgconfig/poppler-cpp.pc
 %{_libdir}/libpoppler.so
 %{_libdir}/libpoppler-cpp.so
-%{_includedir}/poppler
+%dir %{_includedir}/poppler
+%{_includedir}/poppler/cpp/
+# private/unstable
+%{_includedir}/poppler/*.h
+%{_includedir}/poppler/goo/
+%{_includedir}/poppler/fofi/
+%{_includedir}/poppler/splash/
```

Comment:
Instead of avoiding the earlier %{_includedir}/poppler and having a fine-grained list here,the same thing could be achieved by having here

%exclude %{_includedir}/qt5
%exclude %{_includedir}/glib

Reply:
This still.